client/wfprev-war/src/main/angular/src/app/components/edit-project/project-details/project-files/project-files.component.ts

```diff
@@ -253,11 +253,10 @@ export class ProjectFilesComponent implements OnInit {
   
         this.projectService.createActivityBoundary(this.projectGuid, this.fiscalGuid, this.activityGuid, activityBoundary).subscribe({
           next: (boundaryResp) => {
-            const boundaryGuid = boundaryResp?.activityBoundaryGuid;
   
             const attachment: FileAttachment = {
               sourceObjectNameCode: { sourceObjectNameCode: "TREATMENT_ACTIVITY" },
-              sourceObjectUniqueId: boundaryGuid,
+              sourceObjectUniqueId: this.activityGuid,
```

Comment:
this should still be boundaryGuid as you are creating an activity boundary. If you use activityGuid here, the polygon hectares will not be displayed for spatial files. The activity/projectGuid should only be used if there is no spatial related to the file being uploaded, e.g. prescription files